frame/composable-maths/src/labs/numbers.rs

```diff
@@ -201,7 +205,7 @@ impl<T: FixedPointNumber> FixedPointMath for T {
 		//       1           1 | Overflow
 		//    -1/1           0 | DivisionByZero
 		if other.into_inner().is_zero() {
-			return Err(DivisionByZero)
+			return Err(DivisionByZero);
```

Comment:
Curious to know if these were about a clippy warning or a compiler complaint

Reply:
actually it was added automatically, but I don't remember if it was due to `cargo fmt` or `cargo clippy`.